.travis.yml

```diff
@@ -3,7 +3,7 @@ language: go
 
 go:
   - 1.12.x
-  - master
+  # - master
```

Comment:
is master unstable?

Reply:
No, it just takes too long to compile right now :) A lot of activity after 1.13 release, and gimme can't cache it